hw/ip/otbn/dv/model/otbn_model.cc

```diff
@@ -262,7 +264,7 @@ extern "C" int run_model(const char *imem_scope, int imem_words,
 
   std::ostringstream cmd;
   cmd << model_path << " " << imem_words << " " << ifname << " " << dmem_words
-      << " " << dfname << " " << cfname << " " << tfname;
+      << " " << dfname << " " << cfname << " " << tfname << " " << start_addr;
```

Comment:
I feel like this command line is getting a little crazy. Would you be happy if we dumped a configuration file (JSON?) or something and read it in the Python? Obviously, not part of this PR, but it would be good to get your thoughts.

Reply:
I quite enjoy the ability to run a tool without constructing a config file first. But there are still ways to make this easier to use, e.g. named arguments, or environment variables (the standard way of passing data between processes in qmail/daemontools times, I probably spent too much time with them ...)

Reply:
I quite enjoy the ability to run a tool without constructing a config file first. But there are still ways to make this easier to use, e.g. named arguments, or environment variables (the standard way of passing data between processes in qmail/daemontools times, I probably spent too much time with them ...)

Reply:
Yeah, or maybe a default naming scheme would be the right thing. We could pass the name of a directory containing all the stuff, and then it would look at `path/imem`, `path/dmem` etc.